src/firmware/posix/CMakeLists.txt

```diff
@@ -53,11 +53,13 @@ elseif ("${BOARD}" STREQUAL "rpi")
 		-Wl,--end-group
 		)
 
+	file(GLOB RPI_CONFIG_FILES ${PX4_SOURCE_DIR}/posix-configs/rpi/*.config)
 	px4_add_scp_push(OUT upload
 			 OS ${OS}
 			 BOARD ${BOARD}
 			 FILES ${CMAKE_CURRENT_BINARY_DIR}/px4
-				 ${PX4_SOURCE_DIR}/posix-configs/rpi/px4.config
+				${RPI_CONFIG_FILES}
+				${PX4_SOURCE_DIR}/ROMFS
```

Comment:
Yes we only need the mixer files, but the whole directory is small and this also makes directory handling much easier (If I copied only a subdirectory, I would need to setup the directory structure on the target)